### PR TITLE
Md - react issue 1

### PR DESCRIPTION
**Pull request by mertdusunceli**
Completed react issue #1

### CODE REVIEW

File: app/javascript/image_sharing/components/Form.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+export default class Form extends React.Component {
+	constructor(props) {
+		super(props);
+	    this.state = {name: '', comment: ''};
```
Comment (WigginWannabe) [inline app/javascript/image_sharing/components/Form.js]: Please fix indentation and remove empty line at end of function.

File: app/javascript/image_sharing/components/Form.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  	}
+
+  	handleSubmit(event) {
+    	alert('A name was submitted: ' + this.state.name);
+    	event.preventDefault();
+  	}
+
+	render () {
+		return (
+		    <div style={{display: 'flex',  justifyContent:'center', alignItems:'center', height:'100%', margin:'30px'}}>
```
Comment (WigginWannabe) [inline app/javascript/image_sharing/components/Form.js]: There's an extra space before justifyContent, and missing spaces after colons.

File: app/javascript/image_sharing/components/App.js [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
+import Form from './Form';
+import Footer from './Footer';
 
 export default function App() {
   return (
     <div>
       <Header title="Tell us what you think" />
-      Put your components here: Flash Message, Form, Footer
+	  <Form />
+	  <Footer />
```
Comment (WigginWannabe) [inline app/javascript/image_sharing/components/App.js]: Indent these lines to align with Header.

File: app/javascript/image_sharing/test/components/Form.spec.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+/* eslint-env mocha */
+
+import assert from 'assert';
+import { mount } from 'enzyme';
+import React from 'react';
+import Form from '../../components/Form';
+
+describe('<Header />', () => {
```
Comment (WigginWannabe) [inline app/javascript/image_sharing/test/components/Form.spec.js]: Describe Form, not header

**Comment by mertdusunceli** [comment]
[React Issue 1: Add Footer](https://trello.com/c/J9hm2V5p/307-react-issue-1-add-footer)

**Pull request merged**